src/Hadolint/Rule/DL3045.hs

```diff
@@ -85,5 +85,7 @@ quotePredicate c
 
 isWindowsAbsolute :: Text.Text -> Bool
 isWindowsAbsolute path
-  | Char.isLetter (Text.index path 0) && (':' == Text.index path 1) = True
+  | Text.length path >= 2
+    && Char.isLetter (Text.index path 0)
+    && (':' == Text.index path 1) = True
```

Comment:
Is there a version of that function that returns a maybe? We should use that instead

Reply:
Not exactly, but `uncons` is close enough, since this is at the start of `path`.